pom.xml

```
@@ -42,12 +43,24 @@
     </developers>
 
     <scm>
-        <connection>scm:git:git://github.com/jenkinsci/${project.artifactId}-plugin.git</connection>
-        <developerConnection>scm:git:ssh://git@github.com/jenkinsci/${project.artifactId}-plugin.git</developerConnection>
+        <connection>scm:git:https://github.com/jenkinsci/${project.artifactId}-plugin.git</connection>
+        <developerConnection>scm:git:git@github.com:jenkinsci/${project.artifactId}-plugin.git</developerConnection>
         <url>https://github.com/jenkinsci/${project.artifactId}-plugin</url>
       <tag>HEAD</tag>
     </scm>
-    
+
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>io.jenkins.tools.bom</groupId>
+                <artifactId>bom-2.361.x</artifactId>
+                <version>2025.v816d28f1e04f</version>
```

If you are really intending to continue testing and supporting Jenkins versions that supported Java 11, then you should choose the most recent release of the 2.361.x plugin BOM, as noted in the ["depending on older versions"](https://github.com/jenkinsci/bom?tab=readme-ov-file#depending-on-older-versions) documentation.

I think that 2.479.3 is a much better choice, since the Jenkins project has stopped supporting Java 11 a year or more ago.
```suggestion
                <version>2102.v854b_fec19c92</version>
```

---

I absolutely agree with your comments, and I'll address them. I should mention that upgrading from Jenkins 1.x was challenging given the 7+ year gap since the last update. I used AI tools to help navigate the large codebase, but as you correctly noted, the volume of changes is substantial, and I understand it's not ideal to address everything at once.